app/views/pages/you_dont_have_a_trn.html.erb

```diff
@@ -8,7 +8,7 @@
     <h2 class="govuk-heading-l" id="how-to-get-a-trn">How to get a TRN</h2>
     <p class="govuk-body">If you’re eligible for a TRN in England, you should be allocated one automatically by the Teaching Regulation Agency or Capita Teachers’ Pensions.</p>
 
-    <p class="govuk-body">If you do not have a TRN but need one to do a national professional qualification (NPQ), you should get instructions about how to request one. If you do not receive these instructions, email <a class="govuk-link" href="mailto:qts.enquiries@education.gov.uk">qts.enquiries@education.gov.uk</a>.</p>
+    <p class="govuk-body">If you do not have a TRN but need one to do a national professional qualification (NPQ), you should get instructions about how to request one. If you do not receive these instructions, email <a class="govuk-link" href="mailto:teaching.status@education.gov.uk">teaching.status@education.gov.uk</a>.</p>
```

Comment:
```suggestion
    <p class="govuk-body">If you do not have a TRN but need one to do a national professional qualification (NPQ), you should get instructions about how to request one. If you do not receive these instructions, email <%= govuk_link_to t("service.email"), "mailto:#{t("service.email")}" %>.</p>
```

So we don't have to find all cases of the email address if it ever changes again in the future.